src/Dev/SapphireTest.php

```diff
@@ -389,17 +391,19 @@ public static function tearDownAfterClass()
 
         // Reset DB schema
         static::resetDBSchema();
+
+        static::$fixtureFactory = null;
     }
 
     /**
      * @return FixtureFactory
      */
-    public function getFixtureFactory()
+    public static function getFixtureFactory()
     {
-        if (!$this->fixtureFactory) {
-            $this->fixtureFactory = Injector::inst()->create(FixtureFactory::class);
+        if (!static::$fixtureFactory) {
+            static::setFixtureFactory(Injector::inst()->create(FixtureFactory::class));
         }
-        return $this->fixtureFactory;
+        return static::$fixtureFactory;
```

Comment:
Why does this need to be static now? the fixture factory belongs to the class instance. It runs the risk of cross-test pollution.

Reply:
So something interesting that I found out when doing this work is that PHPUnit internally creates a new  object instance for every test method. That means the fixtureFactory instance is not shared from test-to-test on the same class. It has to be a static to allow it to be shared across tests.

The alternative is to have a way of loading a fixture without writing to the DB.